### ISSUE TITLE
Player as Indicator in Panel

### ISSUE THREAD

**Issue by l300lvl** [brianrobles204/Music-Integration#17]
The PLayer as Indicator in Panel option provides options that seem like they should also be available in the Player in Volume Menu option.

For instance in the Indicator mode, e.g indicator in top panel(NOT messagetray), you would see Music Volume, and Shuffle & Repeat toggle switches, whereas in Voume Menu mode you do not.

The Preferences option could also be a button as it appears in the messagetray indicator, but I understand how that part could be confusing.

**Comment by jrub** [comment]
Agree. If "volume menu option" would have shuffle & Reapeat, I would use that, as I prefer the less icons the better.
